### PR TITLE
Freeze frame support

### PR DESCRIPTION
**Pull request by koenvo**
This PR adds support for freeze frames. The implementation is vendor agnostic. This means the freeze frame information can be loaded from any tracking data source. In this PR data is loaded from StatsBomb event data and 360 data. When you want to use a different source you need to think about how to sync the data frame accurate.

Several changes are made:
1. A `freeze_frame` attribute of type `Frame` is added to `Event`
2. The StatsBomb deserializer now loads freeze frames from event data
3. When a 360 feeds is passed to the StatsBomb deserializer the data is attached to the events
4. The Transformer is changed to also transform freeze frames within event data
5. The 'Event data' getting started notebook is extended with a freeze frame example

### CODE REVIEW

File: kloppy/infra/serializers/event/statsbomb/deserializer.py [hunk 1]
```diff
@@ -319,10 +315,64 @@ def _parse_shot(shot_dict: Dict) -> Dict:
...
+                freeze_frame_player["location"], fidelity_version
+            )
+        )
+
+    if event.player not in players_data:
+        # Shot data from event stream might not include coordinates for event player
+        players_data[event.player] = PlayerData(coordinates=event.coordinates)
+
+    return Frame(
+        frame_id=None,  # TODO: come up with a clever value here
```
Comment (JanVanHaaren) [inline kloppy/infra/serializers/event/statsbomb/deserializer.py:365]: How about using the `event_uuid` value, potentially with a prefix or suffix?

**Review by JanVanHaaren — APPROVED** [review]
This pull request looks good to me. Nice addition!

File: kloppy/infra/serializers/event/statsbomb/deserializer.py (hunk 1 above)
Comment (koenvo) [reply kloppy/infra/serializers/event/statsbomb/deserializer.py:365]: Was thinking about it. Now it's of type `int`. Maybe I should calculate it from the timestamp with a assumed fps of 25?

Comment (koenvo) [reply kloppy/infra/serializers/event/statsbomb/deserializer.py:365]: Maybe `(event.period.end_timestamp + event.timestamp) * FPS` with `FPS = 25`

**Pull request merged**